packages/core/package.json

```diff
@@ -84,7 +84,8 @@
     "@kitware/vtk.js": "32.12.1",
     "comlink": "^4.4.1",
     "gl-matrix": "^3.4.3",
-    "loglevel": "^1.9.2"
+    "loglevel": "^1.9.2",
+    "stats.js": "^0.17.0"
```

Comment:
Hmm, I'm not really keen on this. Can we implement it ourselves somehow? I don't like the idea of adding a new dependency just for debugging.